ansible/tasks/internal/optimizations.yml

```diff
@@ -1,39 +1,29 @@
-- name: ensure services are stopped
-  community.general.snap:
```

Comment:
is this removal no longer needed?

Reply:
@div yes, this was no longer present at this point in the build